integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -720,11 +728,11 @@ fn test_resharding_v3_base(params: TestReshardingParameters) {
         params
             .loop_actions
             .iter()
-            .for_each(|action| action(&env.datas, test_loop_data, client_handles[0].clone()));
+            .for_each(|action| action(&env.datas, test_loop_data, client_account_id.clone()));
 
         let clients =
             client_handles.iter().map(|handle| &test_loop_data.get(handle).client).collect_vec();
-        let client = &clients[0];
+        let client: &&near_client::Client = &clients[client_index];
```

Comment:
nit1: Do you need to explicit type? 
nit2: Can you drop one of the `&`?

Reply:
no need for explicit type nor `&` - it can be simplified!